cmd/gapit/common.go

```diff
@@ -186,6 +188,31 @@ func getDevice(ctx context.Context, client client.Client, capture *path.Capture,
 	return nil, log.Err(ctx, nil, "No devices found")
 }
 
+func getDesktopTraceDevice(ctx context.Context, flags GapiiFlags) (bind.Device, error) {
+	if flags.Device == "none" {
```

Comment:
Nit: I find switch statements cleaner than chains of ifs. My personal opinion.

Reply:
Done